modules/core/src/v2/baseCoin.ts

```diff
@@ -604,4 +604,14 @@ export abstract class BaseCoin {
   async getSignablePayload(serializedTx: string): Promise<Buffer> {
     return Buffer.from(serializedTx);
   }
+
+  /**
+   * Returns the base address portion of an address
+   *
+   * @param {String} address - an address
+   * @returns {String} - the base address
+   */
+  getBaseAddress(address: string): string {
```

Comment:
Not all coins have a concept of a base address, so this doesn't belong in `baseCoin`